BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -84,6 +84,11 @@ class FLASH_REGION_TYPE:
     "SHA256_NI"       : 0x0002,
     "SHA384_W7"       : 0x0004,
     "SHA384_G9"       : 0x0008,
+    "X64_M7"          : 0x0020, # SSE3
```

Comment:
Pls add the optimization type used for each hash alg for E9/M7/L9/U8/Y8

Reply:
This is covered in more detail in the README. Should we remove comments here to avoid confusion?